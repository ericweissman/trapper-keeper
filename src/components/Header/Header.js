import React from 'react'
import { Link } from 'react-router-dom'
import addbtnWhite from '../../images/addbtn.svg'

const Header = (props) => {
  const {isLoading} = props
  return(
    <header>
      <h1>TrapperKeeper</h1>
      {
        !isLoading && <Link to='/new-note'>
          <img src={addbtnWhite} />
          <p>Add New Note</p>
        </Link>
      }
    </header>
  )
}

export default Header