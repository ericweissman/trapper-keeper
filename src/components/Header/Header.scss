header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  font-family: $font-primary;
  background-color: $primary;

  h1 {
    margin-left: 35px;
    padding: 15px 0;
    color: $white
  }

  a {
    display: flex;
    align-items: center;
    margin-right: 35px;
    color: $white;
    background-color: $primary-light;
    height: 35px;
    width: 160px;
    border-radius: 20px;
    transition: .1s linear;
    &:hover {
      transform: scale(1.01);
      background-color: $primary-light-hover;
      transition: .1s linear;
    }
  }

  img {
    padding-right: 5px;
    padding-left: 10px;
  }
}

@media only screen and (max-width: 600px) {
  header {
    flex-direction: column;
    h1 {
      margin: 0;
    }
    a {
      margin: 0;
      margin-bottom: 15px;
    }
  }
}