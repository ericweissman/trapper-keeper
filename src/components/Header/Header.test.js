import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'

describe('Header', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})