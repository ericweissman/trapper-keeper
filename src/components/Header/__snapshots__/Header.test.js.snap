// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should have match the correct snapshot 1`] = `
<header>
  <h1>
    TrapperKeeper
  </h1>
  <Link
    replace={false}
    to="/new-note"
  >
    <img
      alt="White add button"
      src="addbtn.svg"
    />
    <p>
      Add New Note
    </p>
  </Link>
</header>
`;
